qoqo_for_braket_devices/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "qoqo_for_braket_devices"
-version = "0.2.5"
+version = "0.2.6"
```

Comment:
Not a big deal, but having amazon-braket-sdk>=1.5.9 in here too would make the dependencies more consistent.

Reply:
Added, thank you!